core/gpio/GPIO_digital_driver.cpp

```diff
@@ -12,63 +12,86 @@
 #include <fstream>
 
 #include "GPIO_digital_driver.h"
+#include "core/logger/Logger.h"
 
 namespace simba {
 namespace core {
 namespace gpio {
 
-GpioDigitalDriver::GpioDigitalDriver() {
-}
+GpioDigitalDriver::GpioDigitalDriver() {}
 
-std::string GpioDigitalDriver::getEndpointPath
-(uint8_t pinNumber, std::string endpoint) {
-    return this->path+std::to_string(pinNumber)+"/"+endpoint;
+GpioDigitalDriver::~GpioDigitalDriver() {
```

Comment:
co się stanie jak aplikacja sie wysypie i nie wykona destruktora ?

Reply:
Nie jest to krytyczne, ale można dodać do skryptu startowego czyszczenie gpio

Reply:
Piny zachowają swoje stany i pliki